ifupdown2/addons/dhcp.py

```diff
@@ -61,6 +61,14 @@ def _up(self, ifaceobj):
             dhcp_wait = policymanager.policymanager_api.get_attr_default(
                 module_name=self.__class__.__name__, attr='dhcp-wait')
             wait = not str(dhcp_wait).lower() == "no"
+            inet6_ll_wait = policymanager.policymanager_api.get_iface_default(module_name=self.__class__.__name__, \
+                ifname=ifaceobj.name, attr='inet6-ll-wait')
+            try:
+                ll_wait_time = int(inet6_ll_wait)
```

Comment:
Can we simplify this and replace ll_wait_time with timeout variable name ? and keep the rest of the below code almost similar to what exists today, with the exception of moving to sleep 1 maybe ?